lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -440,6 +440,7 @@ withStakePool tr idx cfg action =
     bracketTracer' tr "withStakePool" $
         withTempDir tr name $ \dir -> do
             withStaticServer dir $ \url -> do
+                putStrLn $ "Launching static server at url: " <> url
```

Comment:
:grimacing: ... There's litteraly a tracer `tr` in scope that can be used instead of printing to stdout!

Reply:
Fixed d31e78541651f28bc38b04be24f3122c1712e226